mlflow/R/mlflow/R/model.R

```diff
@@ -77,14 +78,15 @@ mlflow_predict_model <- function(model, data) {
 #' can be loaded in R. This method by default searches for a flavor supported by R/MLflow.
 #'
 #' @template roxlate-model-uri
+#' @template roxlate-client
 #' @param flavor Optional flavor specification. Can be used to load a particular flavor in case
 #'        there are multiple flavors available.
 #' @export
 mlflow_load_model <- function(model_uri, flavor = NULL, client = mlflow_client()) {
   model_path <- mlflow_download_artifacts_from_uri(model_uri, client = client)
   supported_flavors <- supported_model_flavors()
   spec <- yaml::read_yaml(fs::path(model_path, "MLmodel"))
-  available_flavors <- intersect(names(spec$flavors), supported_flavors)
```

Comment:
Why did you remove intersection with supported flavors? How do we know which flavors are available in R?

Reply:
Oops good question, accidentally included that change in this PR - it's meant to be included in https://github.com/mlflow/mlflow/pull/1269, will discuss it there & remove the change from this PR